Dockerfile

```
@@ -114,24 +113,25 @@ RUN chown nginx:nginx development.ini
 RUN chmod +x entrypoint_local.sh
 
 # Production setup
-RUN touch production.ini
-RUN touch session-secret.b64
-RUN chown nginx:nginx session-secret.b64
-RUN chown nginx:nginx production.ini
-RUN chown nginx:nginx poetry.toml
+RUN touch production.ini && chown nginx:nginx production.ini && \
+    touch session-secret.b64 && chown nginx:nginx session-secret.b64 && chown nginx:nginx poetry.toml && \
+    touch supervisord.log && chown nginx:nginx supervisord.log && \
+    touch supervisord.sock && chown nginx:nginx supervisord.sock && \
+    touch supervisord.pid && chown nginx:nginx supervisord.pid
```

Do the poetry.toml setting at the start or end, as it breaks up the visual rhythm being in the middle.
Honestly, I would probably put the chowns on their own line, even at the vertical cost of more space (or align them so they all start at the same place on line)? Please double-check the following (untested) for typos:
```suggestion
RUN chown nginx:nginx poetry.toml && \
    touch production.ini && \
    chown nginx:nginx production.ini && \
    touch session-secret.b64 && \
    chown nginx:nginx session-secret.b64 &&  \
    touch supervisord.log && \
    chown nginx:nginx supervisord.log && \
    touch supervisord.sock && \
    chown nginx:nginx supervisord.sock && \
    touch supervisord.pid && \
    chown nginx:nginx supervisord.pid
```